### PR TITLE
Task 1.1+1.2 done

### PR DESCRIPTION
**Pull request by anna2506**
Выполнила задание 1.1, реализовала бинарную и мультиклассовую классификацию. Также реализовала проверку метрики для бинарной и мультиклассовой классификации. Значение k соседей выбрала 5, на финальной тестовой выборке точность составила 0.25.

### CODE REVIEW

**Comment by anna2506** [comment]
@toshiks нет возможности добавить как reviewer

**Comment by anna2506** [comment]
Реализовала линейную классификацию.
Получила следующие результаты:
Accuracy after training for 200 epochs:  0.229
Best validation accuracy achieved: 0.254000
Best learning rate: 0.000100. Best reularization strength: 0.000001
Linear softmax classifier test set accuracy: 0.210000
